packages/paste-customization/src/utils.ts

```diff
@@ -1,6 +1,6 @@
-import type {CustomTheme, CreateCustomTheme} from './types';
+import merge from 'deepmerge';
 
-const merge = require('deepmerge');
+import type {CustomTheme, CreateCustomTheme} from './types';
```

Comment:
Question: I guess my question is any idea why this wasn't getting caught by eslint? It is an eslint rule, so kinda scary it wasn't catching it here

Reply:
Not sure tbh